pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -4046,8 +4046,9 @@ static pj_status_t apply_med_update(pjsua_call_media *call_med,
         if (call->opt.flag & PJSUA_CALL_SET_MEDIA_DIR) {
             call_med->def_dir = call->opt.media_dir[mi];
             PJ_LOG(4,(THIS_FILE, "Call %d: setting audio media "
```

Comment:
It should be replacing the "audio" here?

Reply:
sorry, maybe not, I pushed force again.